src/material/radio/radio.ts

```diff
@@ -639,6 +639,15 @@ export class MatRadioButton
     event.stopPropagation();
   }
 
+  _onLabelClick(event: Event) {
```

Comment:
I don't think we should be fixing this. It's a browser behavior that doesn't really affect the user.

Reply:
mmm, I agree with you actually now.